database/migrations/2025_02_05_152907_create_tasks_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Lightit\Backoffice\Employees\Domain\Models\Employee;
+
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->text('description');
+            $table->string('status');
+            $table->foreignIdFor(Employee::class, 'employee_id');
```

Comment:
```suggestion
            $table
                ->foreignIdFor(Employee::class)
                ->constrained()
                ->cascadeOnDelete();
```
In this case, you can skip passing the column name because Laravel will default to `employee_id`. When you don't pass the column name, it'll automatically give it the name of `[model class name in singular]_id`.

Then I'd advise you to add a constraint to ensure data consistency. Also, it's a good practice to determine what to do if the related model is deleted. In this case, it could be to delete all the tasks related to the employee.